### PR TITLE
Add mock responses for web interface

### PR DESCRIPTION
**Pull request by tonidero**
This adds a new method to the plugin: `setMockWebResults`. If you enable it, you will receive some mocked data when using the plugin on the web. Nothing will be affected in the android/ios versions.

### CODE REVIEW

File: example/purchase-tester/src/components/FunctionTesterContainer.tsx [hunk 1]
```diff
@@ -2,18 +2,19 @@ import './FunctionTesterContainer.css';
...
 import React, {useEffect, useState} from "react";
 import {
-  LOG_LEVEL,
-  PRODUCT_CATEGORY,
   PurchaseDiscountedPackageOptions,
   Purchases
 } from "@revenuecat/purchases-capacitor";
+
 import {REVENUECAT_API_KEY} from "../constants";
+import {LOG_LEVEL, PRODUCT_CATEGORY} from "@revenuecat/purchases-typescript-internal";
```
Comment (vegaro) [inline example/purchase-tester/src/components/FunctionTesterContainer.tsx:10]: Is this change required for everyone? Did you figure out why?

File: ios/Plugin/PurchasesPlugin.swift [hunk 2]
```diff
@@ -43,6 +43,10 @@ public class PurchasesPlugin: CAPPlugin, PurchasesDelegate {
         call.resolve()
     }
 
+    @objc func setMockWebResults(_ call: CAPPluginCall) {
+        call.resolve()
```
Comment (vegaro) [inline ios/Plugin/PurchasesPlugin.swift:47]: We are logging error on Android. Should we log in iOS too?

**Review by vegaro — APPROVED** [review]
Looks good! Just one question about the imports

File: ios/Plugin/PurchasesPlugin.swift (hunk 2 above)
Comment (tonidero) [reply ios/Plugin/PurchasesPlugin.swift:48]: Done in https://github.com/RevenueCat/purchases-capacitor/pull/97/commits/e0ac6606bedbf4b598f6842a561b908b9ada6739

File: example/purchase-tester/src/components/FunctionTesterContainer.tsx (hunk 1 above)
Comment (tonidero) [reply example/purchase-tester/src/components/FunctionTesterContainer.tsx:10]: Haven't figured it out yet :( But we should figure it out before final launch
